src/modules/stackoverflow/stackoverflow.actions.js

```diff
@@ -0,0 +1,32 @@
+import types from "./stackoverflow.types";
+import StackOverFlowApi from "./stackoverflow.api";
+
+const stackApi = new StackOverFlowApi();
+
+export const setThreadsAction = threads => {
+	return {
+		type: types.SET_THREADS,
+		payload: {
+			threads: threads.items,
```

Comment:
Tutaj przekazujesz obiekt z danami. Jeśli  w reducer ma działać to tutaj powinno być `payload.threads.items`

PS. Zobacz najpierw komentarz w `reducer`